assets/style.css

```diff
@@ -0,0 +1,832 @@
+}
+
+.feature-main__top {
+  justify-content: space-between;
+  margin: 20px 0 0 12px;
+}
+
+.news__title1 {
+  font-size: 7.5rem;
+  line-height: 1.1;
+  margin-top: 17rem;
+  flex-wrap: wrap;
+}
+
+.news__article2 {
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(../assets/images/Sprint_image.jpg) center;
+  background-size: cover;
+  grid-area: news__article2;
+  padding: 1rem;
+}
+
+.news__article3 {
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(../assets/images/icon1.png) center;
+  background-size: cover;
+  grid-area: news__article3;
+}
+
+.app-title1 {
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(../assets/images/icon1.png) center;
+  background-size: cover;
+  margin: 10px 10px 0 10px;
+}
+
+.news__title {
+  font-weight: 900;
+  font-size: 4rem;
+  color: #fff;
+  margin-top: 70px;
+  padding-left: 4%;
+}
+
+.news__author {
+  font-size: 2.1rem;
+  font-weight: 400;
+  padding-left: 4%;
+}
+
+.article__author {
+  font-size: 1.9rem;
+  padding-left: 4%;
+  margin-top: 0;
+}
+
+.latestNews {
+  flex-direction: column;
+  align-items: center;
+}
+
+.latestNews h1 {
+  align-self: flex-start;
+  margin: 30px 0 20px 55px;
+}
+
+.latestContainer {
+  display: grid;
+  gap: 2.5rem;
+  grid-template-columns: repeat(4, minmax(100px, 1fr));
+  justify-content: center;
+  padding: 55px;
+}
+
+.latestContainer img {
+  width: 100%;
+  display: block;
+}
+
+.latestContainer a {
+  color: #000;
+  text-decoration: none;
+  font-family: "ARS Maquette", "Helvetica Neue", Arial, sans-serif;
+  font-size: 15px;
+}
+
+.latestItem-title h3 {
+  font-size: 2.5rem;
+  min-height: 8vh;
+  margin-top: 20px;
+}
+
+.latestItem-title strong {
+  color: #aaa;
+}
+
+/* SPONSOR SECTION */
+
+.latest-funding {
+  align-self: flex-start;
+  margin: 3% 0 0 5%;
+}
+
+.latest-funding h2 {
+  justify-content: center;
+}
+
+.funding-round {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 2vh;
+  padding: 3vh;
+}
+
+.funding-links a {
+  padding: 14px 0;
+  color: #f42;
+  font-size: 2rem;
+  position: relative;
+}
+
+.funding-links div {
+  flex-direction: row;
+  border-right: solid 1px #f42;
+  float: left;
+  padding: 0 12px;
+  margin: 12px;
+}
+
+.sponsor-details {
+  display: grid;
+  grid-template-columns: repeat(4 minmax(min-content max-content));
+  font-size: 2rem;
+  color: #000;
+  font-weight: 700;
+  line-height: 1.25;
+}
+
+.sponsor-image img {
+  width: 60px;
+  height: 60px;
+  margin: 10px;
+  position: absolute;
+}
+
+.sponsor-details h3 {
+  background-color: #f42;
+  color: #fff;
+  padding: 10px 16px 10px 84px;
+  font-size: 3rem;
+  margin: 0;
+}
+
+.sponsor-details h5 {
+  font-size: 14px;
+  margin: 0;
+  font-family: Shentox, sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  letter-spacing: 0.04em;
+  line-height: 1.5;
+  color: #000;
+  opacity: 0.5;
+}
+
+.sponsor-assets {
+  padding: 10px 16px 10px 84px;
+  color: #000;
```

Comment:
[Required] Let's use `rem`  instead of `px`. It will be very helpful in responsive design.
[Required] You are using larger font sizes than the original one. Let's try to follow the similar font-size, width, and weight as in the original one.
It should be :point_down: 
![Screenshot from 2020-06-02 15-27-53](https://user-images.githubusercontent.com/37747619/83507640-8b05c300-a4e8-11ea-9b91-734ecd843476.png)

But currently :point_down: 
![Screenshot from 2020-06-02 15-27-43](https://user-images.githubusercontent.com/37747619/83507687-9a850c00-a4e8-11ea-8cfd-d528a7f07947.png)

[Required] Align the main nav and secondary nav properly on the left side.